### ISSUE TITLE
Background is detected

### ISSUE THREAD

**Issue by oheungmin** [zylo117/Yet-Another-EfficientDet-Pytorch#610]
Hello, zylo117. thank you for you code and repo.
I have some question.

I want training mask detection model with one class(mask), using pre-trained model(efficientdet-b1.pth).I am training it for 100 epochs , adamw optmizer with 1000 images as training data , 200 for validation. And all training image are 250 x 250 resolution.
During training and valiation time, it looks very good. In fact, the model detects the target object well. However, in images without target object, the model detects the background as the traget object.

Is it overfitting? What can I try? change anchor option? I wait your answer.
thank you.

**Comment by zylo117** [comment]
I think that's more a underfitting than a overfitting

**Comment by oheungmin** [comment]
Thank you for your reply.
Should I add training data? I plan to retraining with 2000 training data and d0 pre-trained weight.Can you recommended learning rate? 
Thanks once again.

**Comment by zylo117** [comment]
Try tuning it according to the tensorboard graph
